.github/workflows/zxc-build-library.yaml

```diff
@@ -0,0 +1,100 @@
+name: "ZXC: Build Library"
+on:
+  workflow_call:
+    inputs:
+      hapi-version-tag:
+        type: string
+        required: false
+        default: "v0.30.0"
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  hapi-version:
+    name: Version Check
+    runs-on: ubuntu-latest
```

Comment:
Is the `Version Check` only running on ubuntu builds? If so, is that intended?

Reply:
No and yes. This is checking the HAPI repository version tag input parameter to ensure it is a sane and parseable semantic version which begins with the letter `v` prefix.